docs/pages/blog/nextjs-keystone.md

```diff
@@ -0,0 +1,137 @@
+---
+title: "Use Keystone in Next.js applications"
+description: "Learn how to use Keystone in Next.js apps and deploy to Vercel."
+publishDate: "2022-12-14"
+authorName: "Dinesh Pandiyan"
+authorHandle: "https://twitter.com/flexdinesh"
+metaImageUrl: ""
+---
+
+Keystone is an excellent headless CMS that provides a GraphQL API to access your data. But did you also know that you could use **Keystone as a data layer within Next.js applications**? Yes, you don't need to host Keystone on a separate server. Keystone's entire data layer is so well abstracted that you could just import the `context` API in a Next.js application and CRUD your data. In other words, Keystone can be imported and used in serverless environments like Next.js routes, `getStaticProps`, `getServerSideProps` and `next@13` server components.
```

Comment:
>`next@13` server components

Have you checked that? I wouldn't be surprised if it didn't work

Reply:
Uhm no. I've seen examples with prisma so I thought the context API would work as well. To be safer, I will remove this from the post until we test if it will work.